### PR TITLE
Adding note about swap and Nagios.

### PR DESCRIPTION
**Pull request by hyperbolic2346**

### CODE REVIEW

File: pages/k8s/monitoring.md [hunk 1]
```diff
@@ -168,6 +168,36 @@ juju config nrpe nagios_master=<ip-address-of-nagios>
 
 See the [External Nagios][external-nagios] section of the NRPE charm readme for more information.
 
+### Nagios configuration notes
+
+It should be noted that the default Nagios configuration includes monitoring swap.
+Swap is diabled on Kubernetes nodes. Information about this can be found in
```
Comment (tvansteenburgh) [inline pages/k8s/monitoring.md]: Can you link to the place in our charms where swap is disabled? I can't find it. We do however set 'fail-swap-on=false' on kubelet, which leads me to believe that we don't disable swap by default.

Comment (hyperbolic2346) [reply pages/k8s/monitoring.md]: This is a fair point. I was getting errors on AWS, but it seems more an EC2 thing than our code disabling swap. I have swap on a MaaS cluster, so I wouldn't get errors by default on the MaaS cluster. This brings up the question should we disable swap or leave it to the user. Should I reword this to say "if you're doing what Kubernetes suggests with swap, you can disable swap errors with .." or just nuke this PR? I'm happy with either option unless we intend to add swap manipulation to the charms.

Comment (evilnick) [reply pages/k8s/monitoring.md]: can we resolve this yet?
